torch/_inductor/codegen/triton.py

```diff
@@ -3737,11 +3737,12 @@ def store_cache():
             else:
                 # We have to clone the inplace updated arguments to avoid earlier calls
                 # generating out of range indices for later calls.
-                ms = benchmarker.benchmark_gpu(
-                    lambda: call(wrapped_jit_function.clone_args(*args)[0])
+                kernel_callable = lambda: call(  # noqa: E731
+                    wrapped_jit_function.clone_args(*args)[0]
                 )
-                ms_clone = benchmarker.benchmark_gpu(
-                    lambda: wrapped_jit_function.clone_args(*args)[0]
+                clone_callable = lambda: wrapped_jit_function.clone_args(*args)[0]  # noqa: E731
+                ms, ms_clone = benchmarker.benchmark_many_gpu(
```

Comment:
benchmark_many_gpu is eventually going to prune between the choices to find the fastest one (right?). in this case, we're not actually comparing these two for perf and dont expect them to give similar results. Should we benchmark separately ? or am i not understanding this correctly